src/operator/tensor/sample_op.h

```diff
@@ -21,7 +21,6 @@ struct SampleUniformParam : public dmlc::Parameter<SampleUniformParam> {
   float high;
   TShape shape;
   std::string ctx;
-  int dtype;
```

Comment:
It can also be called imperatively like mx.nd.uniform(...). 

and some times backward inference doesn't work